.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Add [eslint-config-codex](https://www.npmjs.com/package/eslint-config-codex) for Eslint.